core/rust/utils/src/account.rs

```diff
@@ -111,8 +110,6 @@ pub fn close_account_raw<'a>(
         .unwrap();
     **src_account_info.lamports.borrow_mut() = 0;
```

Comment:
Nit: if you had time the panicking functions could be replaced:

replace borrow_mut() with try_borrow_mut() and
remove the unwrap()

Reply:
Good call. I couldn't change to a more specific error for numerical overflow without changing the function signature so I picked InvalidRealloc. Maybe in a major version we can add an error argument.

Reply:
ah yeah, i think ProgramError ok too for this lib but yeah can change in the future if we like.  Thank you for the fix!